src/docs/Colors/Colors.tsx

```diff
@@ -118,7 +118,7 @@ const StyledFigure = styled.figure`
 
 const ColorFigure = (props: ColorProps & SuomifiThemeProp) => {
   const { color, keyName } = props;
-  const hslaAsHex = hslaToHex(color);
+  const hslaAsHex = keyName !== 'brandBase' ? hslaToHex(color) : '#003479';
```

Comment:
Should this be commented for future generations? It might be raising some questions 😅

Reply:
Added a comment for this